index.html

```diff
@@ -5,11 +5,94 @@
+    <link rel="stylesheet" type="text/css" href="../src/styles.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
+    <header>
+        <nav class="header-nav">
+          <img src="../images/sumo1.jpg" alt="sumo" class="logo" />
+          <h2>sumo bakery</h2>
+          <input type="checkbox" id="check" />
+          <label for="check" class="check-btn">
+            <i class="fa fa-bars"><p class="menubtn-text">menu</p></i>
+          </label>
+          <ul class="nav-bar">
+            <li><a href="">Home </a></li>
+            <li><a href="">order </a></li>
+            <li><a href="">service </a></li>
+            <li><a href="">cake </a></li>
+            <li><a href="">feedback </a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+          
+        
+        <section  class="hero-area"> 
+            
+                <div class="hero-info">
+                    <h1>HELLO</h1>
+                    <p> INFORMATION</p>
+                </div>
+                <ul class="hero-info">
+
+                    <li class="cake"><img src="../images/cake1.jpg" alt="cake"></li>
+                    <ul class="example-cake">
+                      <li class="cake"><img src="../images/cake2.jpg" alt="cake"></li>
+                      <li class="cake"><img src="../images/cake3.jpg" alt="cake"></li>
+                      <li class="cake"><img src="../images/cake4.png" alt="cake"></li>
+                      <li class="cake"><img src="../images/cake5.jpg" alt="cake"></li>
+                    </ul>
+                    
+                </ul>
+            
+        </section>
+
+        <section class="service-area">
+    
+          <ul class="content-area">
+                <li class="single-service">
+                    <img src="../images/bakery1.jpg" alt="devices">
+                    <p>Internet for all devices</p>
+                    
+                </li>
+                <li class="single-service">
+                  <img src="../images/bakery2.jpg" alt="devices">
+                  <p>Internet for all devices</p>
+                  
+                </li>
+                <li class="single-service">
+                  <img src="../images/bakery3.jpg" alt="devices">
+                  <p>Internet for all devices</p>
+                
+                </li>
+                <li class="single-service">
+                  <img src="../images/bakery4.jpg" alt="devices">
+                  <p>Internet for all devices</p>
+              
+                </li>
+              
+                
+            </li>
+          </ul>
+      </section>
+      </main>
+      
+      <footer>
+        <h3>site map</h3>
```

Comment:
have another look at the wire frame and how you can use flex-box for the footer. How you would expect the mobile-version to deal with footer?